opcua/server/address_space.py

```diff
@@ -515,7 +515,7 @@ def set_attribute_value(self, nodeid, attr, value):
 
         for k, v in cbs:
             try:
-                v(k, value)
+                v(k, value, old)
```

Comment:
though a bit about is and I rest my case :-) make internal subscription remember the necessary data. and maybe put the code in its own class if you can, internal_subscription is already too complex